testsuite/features/step_definitions/command_steps.rb

```diff
@@ -944,6 +941,28 @@
   $server.run(cmd)
 end
 
+When(/^I install "([^"]*)" product on the proxy$/) do |product|
+  out, = $proxy.run("zypper ref && zypper --non-interactive install --auto-agree-with-licenses --force-resolution -t product #{product}")
```

Comment:
Ah, interesting, I did not know we can use something like `out,`

Reply:
another syntax is `out, _rc`, the underscore maning "throw away".